.github/workflows/test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
Probably we can safely remove 3.8

Reply:
Alright! I'll remove it :)

Reply:
Removed in https://github.com/pymc-devs/pymc-bart/pull/109/commits/cc2a3409c34af64aa0a6fa23b97f82b3d3f0bdd0